activiti-spring-resource-loader/src/test/java/org/activiti/spring/resources/DeploymentResourceLoaderTest.java

```diff
@@ -0,0 +1,41 @@
+package org.activiti.spring.resources;
+
+import org.activiti.engine.RepositoryService;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.function.Predicate;
+
+import static org.junit.Assert.*;
+
+public class DeploymentResourceLoaderTest {
+
+    @Test
+    public void rename_me() {
```

Comment:
You missed the test renaming :) and it looks like not finished yet.

Reply:
enanched test, with a little help of some stubs/mocks